common/views/components/SearchFilters/SearchFilters.js

```diff
@@ -46,12 +13,25 @@ type Props = {|
   changeHandler: () => void,
 |};
 
+export type SearchFiltersSharedProps = {|
+  ...Props,
+  inputDateFrom: string,
+  inputDateTo: string,
+  setInputDateFrom: () => void,
+  setInputDateTo: () => void,
+  workTypeFilters: string[],
+  productionDatesFrom: ?string,
+  productionDatesTo: ?string,
+  workTypeInUrlArray: string[],
+|};
+
 const SearchFilters = ({
   searchForm,
   searchParams,
   workTypeAggregations,
   changeHandler,
 }: Props) => {
+  const [isMobile, setIsMobile] = useState(false);
```

Comment:
It'd be good to make this a hook too?

Similar to what we should do with `const isEnhanced = `useIsEnhanced()`

Reply:
Actually scrap that - I thought this was going to be used for functionality - but it's used for styling.

[This?](https://github.com/wellcometrust/wellcomecollection.org/pull/4948/files#r360421941)